sass/components/Status/status.scss

```diff
@@ -6,3 +6,7 @@
   padding: 0;
   margin: 0;
 }
+
+.kubevirt-status__span {
```

Comment:
can you rename it to `kubevirt-status__popover` ? Otherwise LGTM, I will push it after the change :)

Reply:
:beers: